crates/wdk-build/rust-driver-makefile.toml

```diff
@@ -177,13 +180,16 @@ args = [
   "1.33",
 ]
 
+
 [tasks.infverif]
 private = true
 dependencies = ["stampinf"]
+condition = { env_not_set = ["WDK_INF_NEW_VERSION"] }
 command = "infverif"
 args = [
   "/v",
   "/w",
+  "@@decode(WDK_IS_SAMPLE,true,${WDK_INFVERIF_SAMPLE_FLAG})",
```

Comment:
I think we should not be adding extra vars here specific to samples logic, when its evaluated for all drivers. sample drivers should locally override `WDK_BUILD_ADDITIONAL_INFVERIF_FLAGS`